### ISSUE TITLE
Tests failed:  Expression: all((1 * u"turn" ≈ q * u for (q, u) = zip(quantities, units)))

### ISSUE THREAD

**Issue by Hellseher** [yakir12/UnitfulAngles.jl#38]
Hi,

During preparing your project for Guix downstream I've faced with failing tests:

```
Test Failed at /gnu/store/y0jimlwdn54jkqiicdwsjlp0jbb7marz-julia-unitfulangles-0.7.1/share/julia/loadpath/UnitfulAngles/test/runtests.jl:9
  Expression: all((1 * u"turn" ≈ q * u for (q, u) = zip(quantities, units)))
ERROR: LoadError: There was an error during testing
in expression starting at /gnu/store/y0jimlwdn54jkqiicdwsjlp0jbb7marz-julia-unitfulangles-0.7.1/share/julia/loadpath/UnitfulAngles/test/runtests.jl:9
```
- Julia v1.8.3
- UnitfulAngles.jl v0.7.1
- Unitful.jl v1.12.2

**Comment by cmichelenstrofer** [comment]
Ah, this was introduced by #34 and I missed it when I fixed the `pas` issue in dcc079ab51d9ffa2003f8e950efda02bc9b27788. 

It should be fixed now (v0.7.2), @Hellseher can you verify it works for you if you update to v0.7.2? 

This points to a larger issue that the tests are not checked as part of CI. The GitHub workflows in this repo are outdated. I just created a new issue (#39) for this.

**Comment by Hellseher** [comment]
Hi @cmichelenstrofer,

Thanks for so quick response and fix, it's working like a charm now. Now all
phases are passed with tests enabled and patch is preparing for review to be
included in Guix.

Thanks,
Oleg
